build.gradle

```diff
@@ -4,10 +4,13 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.1"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.gms:google-services:4.3.4'
+        classpath 'com.facebook.android:facebook-android-sdk:[4,5)'
```

Comment:
is the [4,5) a typo? I haven't seen that before

Reply:
https://developers.facebook.com/docs/android/componentsdks/

They had that format and I thought it was weird too. To be honest, the facebook login is still kinda finicky and I've always thought it was that line.